jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/xml/ServicesProcessDataEventListener.java

```diff
@@ -126,7 +129,21 @@ public void onNodeAdded(Node node) {
             } else {
                 processDescriptor.getReusableSubProcesses().add(processId);
             }
+        } else if (node instanceof TimerNode) {
```

Comment:
what about timer start events? do we want to cover them? Mainly in context of event subprocess...

Reply:
@mswiderski makes sense to me, I included the _StartNode_ as well, thanks.